bzr/meta.yaml

```diff
@@ -0,0 +1,18 @@
+package:
+  name: bzr
+  version: 2.6.0
+
+source:
+  fn: bzr-2.6.0.tar.gz
+  url: https://launchpad.net/bzr/2.6/2.6.0/+download/bzr-2.6.0.tar.gz
+  md5: 28c86653d0df10d202c6b842deb0ea35
+
+build:
+  number: 1
```

Comment:
It's not that serious, but all new packages starts with build number 0. It's just how things work :-)